api/axis.py

```diff
@@ -469,6 +471,7 @@ def fulfill(self, patron, pin, licensepool, internal_format, **kwargs):
                 if patron_loan:
                     patron_loan.external_identifier = loan.external_identifier
 
+            fulfillment.can_cache_manifest = True
```

Comment:
I think it's better to set `can_cache_manifest` right at the point where you set `content` to a cacheable value. That would be in `Axis360FulfillmentInfo.do_fetch`. Right you you should only do this for AxisNow manifests,which means `if manifest.MEDIA_TYPE==AxisNowManifest.MEDIA_TYPE`.